From 329b598044d2dfe53774e3608b1b3e92175ebde4 Mon Sep 17 00:00:00 2001
From: FiniteReality <digpoealternate@gmail.com>
Date: Mon, 2 Apr 2018 22:49:54 +0100
Subject: [PATCH 1/1] Use more supported TLS method instead

---
 src/Node.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Node.cpp b/src/Node.cpp
index 70e7961..662eeec 100644
--- a/src/Node.cpp
+++ b/src/Node.cpp
@@ -55,7 +55,7 @@ Node::Node(int recvLength, int prePadding, int postPadding, bool useDefaultLoop)
         nodeData->preAlloc[i] = nullptr;
     }
 
-    nodeData->clientContext = SSL_CTX_new(SSLv23_client_method());
+    nodeData->clientContext = SSL_CTX_new(TLS_client_method());
     SSL_CTX_set_options(nodeData->clientContext, SSL_OP_NO_SSLv3);
 }
 
-- 
2.16.2

